Samples/Server/Unity-FractalStream/Assets/Scripts/WebRTCServer.cs

```diff
@@ -116,19 +137,42 @@ public class WebRTCServer : MonoBehaviour
         /// </summary>
         private bool? lastSetupVisibleEyes = null;
 
+        /// <summary>
+        /// The last prediction timestamp.
+        /// </summary>
+        private long lastTimestamp = -1;
+
+        /// <summary>
+        /// Flag indicating if the camera needs to be updated.
+        /// </summary>
+        private bool cameraNeedUpdated = false;
+
         /// <summary>
         /// Unity engine object Awake() hook
         /// </summary>
         private void Awake()
         {
-            // make sure that the render window continues to render when the game window does not have focus
+            // Make sure that the render window continues to render when the game window 
+            // does not have focus
             Application.runInBackground = true;
+            Application.targetFrameRate = 60;
 
-            // setup the eyes for the first time
-            SetupActiveEyes();
+            // Set screen resolution.
+            Screen.SetResolution(VideoFrameWidth, VideoFrameHeight, false);

```

Comment:
imo this isn't the concern of the plugin, it's up to the caller to ensure their experience is sized as they wish, and they communicate that size information to us via VideoFrameWidth and VideoFrameHeight. removing it from this code also has the advantage of allowing the caller to choose to not encode the entire screen (and to instead use any framebuffer)

Reply:
The video frame size is currently defined in the sample (WebRTCServer.cs) not the plugin right? I had some trouble selecting 2560x720 resolution in the dialog before but I think that we can just select 1280x720 instead and the plugin handles it automatically.

Reply:
I think that the video frame size is an important part of our plugin logic since our hololens clients must match the exact resolution. Otherwise, it can't render. Let's discuss this issue offline to see how we should address it.

Reply:
we can talk offline, sure. my point is that video frame size is not necessarily equal to application screen size in all scenarios. i'd like us to do as little as possible to force those two to be directly tied together within our plugin, and this line of code directly couples those.

Reply:
Oh, I think that I understand your point now but let's discuss tmr.